server/bin/pbench-trampoline

```diff
@@ -6,14 +6,14 @@ _basename=$(basename ${0})
 if [[ ! ":${PATH}:" =~ ":${_dirname}:" ]]; then
    PATH=${_dirname}${PATH:+:}${PATH}; export PATH
 fi
-if command -v python3 > /dev/null 2>&1 ;then
+if command -v python3.9 > /dev/null 2>&1 ;then
     _pythoncmd=""
```

Comment:
If the `python3.9` command merely _exists_, then we don't (necessarily) use it...we fall back on whatever the shebang in the Python script matches?  And, otherwise, we error out...which means that, when control reaches line 20 (or 24) `_pythoncmd` will always be empty, so what's the point of defining and referencing it?

Reply:
All of this pre-python integration.  I don't think it is worth changing now.